django/thunderstore/api/cyberstorm/views/package_version_list.py

```diff
@@ -27,4 +27,4 @@ def get_queryset(self):
             name__iexact=self.kwargs["package_name"],
         )
 
-        return package.versions.active()
+        return package.available_versions
```

Comment:
I'd argue this should call the `package.public_list()` rather than the more expensive `available_versions` query; filtering should be the same for both (at least once my comments are accounted for) and it reflects the intended API for querying in accordance to visibility rules better (ideally no call-site specific logic needed, everything should be able to use the `VisibilityQuerySet` provided methods directly)